server/controllers/config.js

```diff
@@ -0,0 +1,254 @@
+import mysql from "mysql";
+
+import { queryPromise } from "../db/database.js";
+import HttpResponseWrapper from "../global/HttpResponseWrapper.js";
+import { addErrorTitle } from "../global/Logger.js";
+
+import { getAllQuestionTypes, createGeneratorOfType, NotEnoughDataError } from "./question.js";
+
+export const DEFAULT_CONFIG = {
+  ROUNDS_PER_DUEL: 5,
+  QUESTIONS_PER_ROUNDS: 5,
+  QUESTION_TIMER_DURATION: 10,
+};
```

Comment:
Should we instead store the default config in the database at first startup? Then the database will be our single source of trust for these values :smile:

Reply:
Yes we can ! This was just to return valid values ​​in case the database doesn't contain (valid) values, but we can choose to return an error if the data is bad !